code/modules/mob/living/basic/guardian/guardian_creator.dm

```diff
@@ -9,11 +9,13 @@ GLOBAL_LIST_INIT(guardian_radial_images, setup_guardian_radial())
 		option.info = span_boldnotice(initial(guardian_path.creator_desc))
 		.[guardian_path] = option
 
+	/* ONGOING VAMPIRE REWORK, FIX THIS ONCE YOU KNOW WTF TO DO WITH IT
```

Comment:
I mean, others not getting it is because it was added as a jojo reference.

Reply:
Likely going to fix it rather than kill it.